<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="frame_range_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Range:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="frame_range_start">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="frame_range_end">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QLabel" name="file_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name_input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="export_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export As Alembic</string>
    </property>
   </widget>
   <widget class="QLabel" name="root_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Root:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="root_input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="file_choice_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose the root folder where the export will be stored.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Choose Root Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="object_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Object Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="object_name_input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="update_obj_name_btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Updates the object name field, which will be used as the root of the export, with the latest selected object, if multiple objects are selected in the scene only the latest selected object is put.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Update Object Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>file_choice_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>choose_file_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>choose_file_clicked()</slot>
 </slots>
</ui>
